src/Resources/views/Collector/statsd.html.twig

```diff
@@ -19,9 +19,9 @@
 
 {% block menu %}
 <span class="label">
-    <span class="icon">{% spaceless %}
+    <span class="icon">{% apply spaceless %}
```

Comment:
please simply remove the `spaceless` tag here and manually remove all the spaces between HTML tags instead 
(using `apply` introduces breaking changes with twig)

```suggestion
    <span class="icon"><img ...
```